web-ui/src/components/reviews/TeamReviews.jsx

```diff
@@ -122,12 +127,13 @@ const ReviewStatus = {
 
 const TeamReviews = ({ onBack, periodId }) => {
   const { state, dispatch } = useContext(AppContext);
-  const history = useHistory();
```

Comment:
We don't need to use the History API to go back. We just need to call the `onBack` function that is passed in.